monkey/infection_monkey/dropper.py

```diff
@@ -110,27 +111,18 @@ def start(self):
 
                 return False
 
-        if WormConfiguration.dropper_set_date:
-            if sys.platform == "win32":
-                dropper_date_reference_path = os.path.expandvars(
-                    WormConfiguration.dropper_date_reference_path_windows
-                )
-            else:
-                dropper_date_reference_path = WormConfiguration.dropper_date_reference_path_linux
+        if sys.platform == "win32":
+            dropper_date_reference_path = os.path.expandvars(DATE_REFERENCE_PATH_WINDOWS)
+        else:
+            dropper_date_reference_path = DATE_REFERENCE_PATH_LINUX
             try:
                 ref_stat = os.stat(dropper_date_reference_path)
+                os.utime(self._config["destination_path"], (ref_stat.st_atime, ref_stat.st_mtime))
             except OSError:
                 logger.warning(
                     "Cannot set reference date using '%s', file not found",
                     dropper_date_reference_path,
                 )
```

Comment:
The logic has changed here slightly, the log message should change accordingly.

Reply:
I think the `try ... except` block has wrong indentation, that's what Shreya wants to tell

Reply:
That, too.
But I also mean that an exception can be raised because of line 119, which the current log message covers, but also because of line 120, which the current log message does not cover, which was covered previously in the separate try/except block.